src/main/java/ro/chirila/programarispital/controller/TypeOfServiceController.java

```diff
@@ -0,0 +1,53 @@
+package ro.chirila.programarispital.controller;
+
+import jakarta.transaction.Transactional;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ro.chirila.programarispital.repository.dto.TypeOfServiceRequestDTO;
+import ro.chirila.programarispital.repository.dto.TypeOfServiceResponseDTO;
+import ro.chirila.programarispital.service.TypeOfWorkService;
+
+import java.util.List;
+
+@CrossOrigin(origins = "*", maxAge = 3600)
+@RestController
+@RequestMapping("/service")
+public class TypeOfServiceController {
+    private final TypeOfWorkService typeOfWorkService;
+
+    public TypeOfServiceController(TypeOfWorkService typeOfWorkService) {
+        this.typeOfWorkService = typeOfWorkService;
+    }
+
+    @PostMapping("/add")
+    public ResponseEntity<TypeOfServiceResponseDTO> addService(@RequestBody TypeOfServiceRequestDTO typeOfServiceRequestDTO) {
+        try {
+            TypeOfServiceResponseDTO response = typeOfWorkService.addService(typeOfServiceRequestDTO);
+            return new ResponseEntity<>(response, HttpStatus.CREATED);
+        } catch (RuntimeException e) {
+            return new ResponseEntity<>(null, HttpStatus.BAD_REQUEST);
+        }
+    }
+
+    @Transactional
+    @GetMapping("/get-all-services")
+    public ResponseEntity<List<TypeOfServiceResponseDTO>> getAllServices() {
+        return new ResponseEntity<>(typeOfWorkService.getAllTypeOfServices(), HttpStatus.OK);
+    }
+
+    @GetMapping("/getService")
```

Comment:
keep consistency naming for endpoints. Here, use the default `/` instead of `getService` and below `doctors-by-service`